### PR TITLE
[FIX] crm_lead_vat: fix onchange error

### PR DESCRIPTION
**Pull request by oihane**
capturadepantallade2017-06-16115715

This is the fixed error.

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Review by agaldona — APPROVED** [review]

File: crm_lead_vat/models.py [hunk 1]
```diff
@@ -22,9 +22,10 @@ def _lead_create_contact(self, lead, name, is_company, parent_id=False):
     def on_change_partner_id(self, partner_id):
```
Comment (yajo) [inline crm_lead_vat/models.py:22]: Although the fix is good, I feel the real fix to your error would be adding `@api.multi` for this method...

**Review by yajo — APPROVED** [review]

**Comment by yajo** [comment]
Check the bots please.

File: crm_lead_vat/models.py (hunk 1 above)
Comment (oihane) [reply crm_lead_vat/models.py:22]: That was the first option, but as it is an old onchange it does not work as expected. So this was the real fix.

**Comment by oihane** [comment]
@Yajo it's already green

**Pull request merged**